src/zjs_callbacks.c

```diff
@@ -420,35 +420,40 @@ void signal_callback_priv(zjs_callback_id id,
 {
     DBG_PRINT("pushing item to ring buffer. id=%d, args=%p, size=%lu\n", id,
               args, size);
-
-    if (GET_TYPE(cb_map[id]->flags) == CALLBACK_TYPE_JS) {
-        // for JS, acquire values and release them after servicing callback
-        int argc = size / sizeof(jerry_value_t);
-        jerry_value_t *values = (jerry_value_t *)args;
-        for (int i=0; i<argc; i++) {
-            jerry_acquire_value(values[i]);
+    if (id >= 0 && cb_map[id]) {
```

Comment:
How about change this to
```
if (id < 0 || !cb_map[id]) {
   // some error
   return;
}

Then that odd situation will get a dbg print or something, instead of being ignored... and the indentation won't change so much so easier to review. :)